docs/root/intro/version_history.rst

```diff
@@ -1,6 +1,10 @@
 Version history
 ---------------
 
+1.14.2 (June 6, 2020)
```

Comment:
Please do a master PR to merge this version history in once you finalize the date.

Reply:
Done: #11495.